JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -1,10 +1,34 @@
-This demo illustrates the Popup component. Click the Details button under an employee picture to see what the Popup looks like. Here is how you can configure its main elements:
+This demo shows how to show and hide the Popup component, how to fill it with content, specify its position and other configurations. To get started with the DevExtreme Popup component, refer to the following tutorial for step-by-step instructions: [Getting Started with Popup](/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/).
 
-- **Content**        
-Put any valid markup inside the component. 
+## Show and Hide the Popup
 
-- **Title Bar**       
-Set [showTitle](Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) to **true** and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to specify the caption. 
+Use two-way binding between the [visible](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#visible) property and a Boolean variable (`popupVisible` in this demo). The `showInfo(employee)` method sets `popupVisible` to `true` and the Popup appears. You can close the Popup in multiple ways:
```

Comment:
Hi Vlada, :) I know in your mind they are related, but when I am reading - they are not. In fact, your explanation further confuses me, because of how much it differs from the language in the document. Talking about two-way binding is not the same as telling our customer that they need to bind the visible property to a variable. Can you please rewrite this whole section so that those ideas are clear and those sentences logically flow into each other. Thanks.

Reply:
Okay, I will rewrite it